adoc/retail_chap_next.adoc

```diff
@@ -61,10 +61,10 @@ If you need further information or support, try one of these options.
 
 For {susemgr} documentation, visit https://www.suse.com/documentation/suse-manager-3/
 
-For legacy {slepos} docuemntation, visit https://www.suse.com/documentation/slepos11/index.html
-Note, however, that {smr} documentation supercedes this information.
+For legacy {slepos} documentation, visit https://www.suse.com/documentation/slepos11/index.html
+Note, however, that {smr} documentation supersedes this information.
 
 [[retail.sect.next.support]]
 === Support
 
-For personalised support, log in to your {scc} account here: https://scc.suse.com/login
+For personalized support, log in to your {scc} account here: https://scc.suse.com/login
```

Comment:
I do not like those constructs neither:

``... foo bar here: https://baz.html``

If the URL is really long finish the sentence with a colon and put the URL in line of its own.  If it is short, a normal sentence will probably do.